src/Xhgui/Profile.php

```diff
@@ -398,6 +398,49 @@ public function sort($dimension, $data)
         uasort($data, $sorter);
         return $data;
     }
+    
+    /**
+     * @param array $profileData
+     * @param array $filters
+     *
+     * @return array
+     */
+    public function filter($profileData, $filters = [])
+    {
+        $filter = function ($item, $key) use ($filters) {
+            $nameToFilter = $this->getNameToFilter($key);
+            return !in_array($nameToFilter, $filters);
+        };
+        
+        $profileData = array_filter($profileData, $filter, ARRAY_FILTER_USE_BOTH);
+        foreach ($filters as $key => $item) {
+            foreach ($profileData as $key_item => $method) {
+                if (fnmatch($item, $key_item)) {
```

Comment:
Variables are generally using camelBacked casing in xhgui.

Reply:
sorry, I am used the snakeCase because I work a lot on WordPress.